src/components/LinkChooser/SearchInput.tsx

```diff
@@ -1,154 +1,112 @@
-            onBlur,
-            onClear,
-            onClick,
-            size,
             machineService,
+            onClear,
+            validation = Validation.Default,
         },
         inputElement,
     ) => {
-        const { isFocusVisible, focusProps } = useFocusRing({ within: true, isTextInput: true });
-        const { isFocusVisible: previewButtonIsFocusVisible, focusProps: previewButtonFocusProps } = useFocusRing();
-        const { isFocusVisible: copyButtonIsFocusVisible, focusProps: copyButtonFocusProps } = useFocusRing();
-        const { isFocusVisible: clearButtonIsFocusVisible, focusProps: clearButtonFocusProps } = useFocusRing();
+        const { value } = ariaProps;
+        const { isFocusVisible, focusProps } = useFocusRing({ isTextInput: true });
 
         const [, send] = useActor(machineService);
 
-        const onKeyDown = (event: KeyboardEvent<HTMLInputElement>) => {
-            if (event.key === "Enter") {
-                onEnterPressed && onEnterPressed(event);
-            }
-        };
+        const isLoading = validation === Validation.Loading;
 
         return (
             <div
-                {...focusProps}
                 data-test-id="link-chooser-search-wrapper"
                 className={merge([
-                    "tw-flex tw-items-center tw-h-9 tw-gap-2 tw-px-3 tw-border tw-rounded tw-text-s tw-font-sans tw-relative tw-bg-white dark:tw-bg-transparent",
-                    dotted ? "tw-border-dashed" : "tw-border-solid",
+                    "tw-flex tw-items-center tw-h-9 tw-gap-2 tw-px-2.5 tw-border tw-rounded tw-text-s tw-font-sans tw-relative tw-bg-white dark:tw-bg-transparent",
                     disabled
                         ? "tw-border-black-5 tw-bg-black-5 dark:tw-bg-black-90 dark:tw-border-black-90"
                         : merge([
-                              "focus-within:tw-border-black-90",
-                              validationStyle[validation],
+                              "focus-within:tw-border-black-100",
                               isFocusVisible && FOCUS_STYLE,
+                              validationClassMap[validation],
                           ]),
                 ])}
             >
                 {decorator && (
                     <div
                         className={merge([
-                            "tw-flex tw-items-center tw-justify-center tw-pl-1",
-                            disabled ? "tw-text-black-60" : "tw-text-black-80",
+                            "tw-flex tw-items-center tw-justify-center",
+                            disabled || !selectedResult ? "tw-text-black-40" : "tw-text-violet-60",
                         ])}
                         data-test-id="link-chooser-decorator-icon"
                     >
                         {decorator}
                     </div>
                 )}
                 <input
-                    id={useMemoizedId(propId)}
+                    id={useMemoizedId()}
```

Comment:
I don't think it is needed anymore as it should be contained in `ariaProps`

Reply:
removed